pom.xml

```diff
@@ -476,7 +476,7 @@
 				<plugin>
 					<groupId>com.google.code.sortpom</groupId>
 					<artifactId>maven-sortpom-plugin</artifactId>
-					<version>2.3.0</version>
+					<version>2.4.0</version>
```

Comment:
While it is a good idea to update dependencies, this is not related to your PR (as far as I can see), so it should be part of another PR.